features/geometrisch-zoeken.feature

```diff
@@ -0,0 +1,327 @@
+
+    Intersects:
+    Om te bepalen of een geometrie A binnen een geometrie B ligt, deze raakt, kruist of (deels) overlapt, wordt gebruik gemaakt van de spatial operator INTERSECTS:
+
+      intersects(geometrie A, geometrie B)
+
+    Intersects betekent dat geometrie A en geometrie B tenminste één gemeenschappelijk punt hebben.
+    Anders gezegd geometrie A en geometrie B mogen niet volledig los van elkaar staan.
+    Het resultaat van intersects(geometrie A, geometrie B) levert hetzelfde op als intersects(geometrie B, geometrie A).
+    Het maakt niet uit of geometrie A of geometrie B de geometrie is waarmee gezocht wordt en
+    het maakt niet uit of geometrie A of geometrie B de resource geometrie is.
+    Als intersects true oplevert, dan wordt de resource waar de geometrie onderdeel van is, geretourneerd.
+
+    Referenties:
+    - Haal Centraal Common - foutafhandeling.feature
+    
+    Het geometrisch zoeken met een bounding box geometrie kan met de parameter: bbox.
+    
+    Het geometrisch zoeken met een polygoon geometrie kan met de parameter: polygoon.
+
+    Het geometrisch zoeken met een punt geometrie kan met de parameter: locatie.
+    Het is echter ook mogelijk om een bounding box op te geven waarbij de coördinaten van de linker onderhoek en de rechter bovenhoek exact gelijk zijn.
+
+    In de hierna volgende voorbeelden is een vereenvoudigde representatie gebruikt voor coördinaten.
+    
+    ![Zoeken met locatie voorbeelden](../images/locatie-voorbeelden.png)    
+    ![Zoeken met bounding box voorbeelden](../images/bbox-voorbeelden.png)    
+    ![Zoeken met polygoon voorbeelden](../images/locatie-voorbeelden.png)      
+
+  Achtergrond:
+
+    Gegeven de volgende verzameling van objecten
+
+    | identificatie | geometrie                          |
+    |---------------|------------------------------------|
+    | 0001          | (0,0),   (10,0),  (10,10), (0,10)  |
+    | 0002          | (12,0),  (22,0),  (22,10), (12,10) |
+    | 0003          | (12,12), (22,12), (22,22), (12,22) |
+    | 0004          | (0,8),   (10,8),  (10,18), (0,18)  |
+    
+  Scenario: GET met locatie, Content-Crs (optioneel) en Accept-Crs (optioneel) één resource
+  
+    Als er met de volgende parameters wordt gezocht:
+    | locatie | Content-Crs | Accept-Crs |
+    |---------|-------------|------------|
+    | (5,5)   | -           | -          |
+    | (5,5)   | EPSG:28992  | -          |
+    | (5,5)   | -           | EPSG:28992 |
+    | (5,5)   | EPSG:28992  | EPSG:28992 |
+    | (10,5)  | -           | -          |
+    | (10,5)  | EPSG:28992  | -          |
+    | (10,5)  | -           | EPSG:28992 |
+    | (10,5)  | EPSG:28992  | EPSG:28992 |
+    En de provider ondersteund het CRS in Content-Crs
+    En de provicer ondersteund het CRS in Accept-Crs
+    En de resource geometrie intersects met de opgegeven bounding box geometrie
```

Comment:
Wat wordt hier precies beschreven? Hier lees ik resource en in de achtergrond objecten. Ook wordt hier gesproken over bounding box, maar in de als en de scenario titel zie ik locatie.
Wat ook heel verwarrend werkt, is dat dit niet conform Gherkin syntax is

Reply:
Zijn dit allemaal punten met een andere Content-crs en accept-crs combinatie? Maar hoe weet ik dan welke van deze parameter combinatie verantwoordelijk is voor de hit (het gevonden object 0001)?